gnomad/utils/release.py

```diff
@@ -91,4 +97,15 @@ def _get_index(label_groups):
             {**_get_index(dict(downsampling=downsamplings, group=["adj"], pop=pops))}
         )
 
+    if additional_strata:
+        for strata in additional_strata:
+            for k, v in strata.items():
+                # Note: Tack the new strata onto the end of the sort order so the labels
+                # can be made
+                if k not in sort_order:
+                    sort_order.append(
+                        k
+                    )  # TODO: Should we add the additional strata before group, aka adj? sort_order.insert(-1,k)
```

Comment:
I think the GATK version looks better after adj, number like 4.1.0.1 in the middle seems a bit weird.